api/hypershift/v1beta1/hostedcluster_conditions.go

```diff
@@ -101,6 +101,11 @@ const (
 	// A failure here may require external user intervention to resolve. E.g. oidc was deleted out of band.
 	ValidOIDCConfiguration ConditionType = "ValidOIDCConfiguration"
 
+	// ValidIDPConfiguration indicates if the Identity Provider configuration is valid.
```

Comment:
Please include this in the list validated by validateHostedClusterConditions e2e

Reply:
I will add this in a follow on.  If I add it now, the upgrade test fails as we check the conditions on the n-1 release before we upgrade to the release under test.